### PR TITLE
fix: Fix webpack 5 usage with mini-css-extract-plugin

### PR DESCRIPTION
**Pull request by bebraw**
It looks like if the plugin is used with mini-css-extract-plugin in webpack 5, `tap` can trigger multiple times. It should trigger only once.

Possibly it's not a bug in **mini-html-webpack-plugin** but it's good to fix anyhow.

### CODE REVIEW

File: example/src/main.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+	background: orange;
```
Comment (sapegin) [inline example/src/main.css:2]: 🍊

File: src/index.ts [hunk 2]
```diff
@@ -221,20 +221,25 @@ class MiniHtmlWebpackPlugin implements WebpackPluginInstance {
...
 			compiler.hooks.compilation.tap(pluginName, (compilation) => {
-				compilation.hooks.processAssets.tapPromise(
-					{
-						name: pluginName,
-						// https://github.com/webpack/webpack/blob/master/lib/Compilation.js#L3280
-						stage: Compilation.PROCESS_ASSETS_STAGE_ADDITIONAL,
-					},
-					() => this.webpack5plugin(compilation)
-				);
+				if (!registeredAlready) {
```
Comment (sapegin) [inline src/index.ts:231]: Nit: early return might be a good idea here ;-)

File: test/__snapshots__/plugin.test.ts.snap [hunk 3]
```diff
@@ -143,3 +143,17 @@ exports[`default options 1`] = `
     </body>
   </html>"
 `;
+
+exports[`mini-css-extract-plugin 1`] = `
```
Comment (sapegin) [inline test/__snapshots__/plugin.test.ts.snap:147]: I don't see anything specific to the CSS plugin here, do we need this snapshot?

File: test/plugin.test.ts [hunk 4]
```diff
@@ -51,6 +52,30 @@ test('default options', async () => {
...
+			rules: [
+				{
+					test: /\.css$/,
+					use: [MiniCssExtractPlugin.loader, 'css-loader'],
+				},
+			],
+		},
+		plugins: [
+			new MiniHtmlWebpackPlugin({}),
+			// @ts-ignore: MCEP TS declarations are broken
```
Comment (sapegin) [inline test/plugin.test.ts:69]: Save couple of brain cycles:

```suggestion
			// @ts-ignore: MiniCssExtractPlugin types are broken
```

File: src/index.ts (hunk 2 above)
Comment (bebraw) [reply src/index.ts:231]: Sure.

File: test/__snapshots__/plugin.test.ts.snap (hunk 3 above)
Comment (bebraw) [reply test/__snapshots__/plugin.test.ts.snap:147]: Yeah, I updated the test to include some CSS.

**Pull request merged**